hummingbird/ml/supported.py

```diff
@@ -64,6 +77,14 @@ def _build_sklearn_operator_list():
             HistGradientBoostingRegressor,
             RandomForestClassifier,
             RandomForestRegressor,
+            # Linear-methods
+            LinearRegression,
+            LinearSVC,
+            LogisticRegression,
+            LogisticRegressionCV,
+            NuSVC,
```

Comment:
can we put `NuSVC` and `SVC` together and under a `#SV` comment? (They are not linear methods)

Reply:
done